### PR TITLE
Fix/lab revisions

### PR DESCRIPTION
**Pull request by EllieShivers**

### CODE REVIEW

File: input/pagecontent/StructureDefinition-cbs-specimen-intro.md [hunk 1]
```diff
@@ -1,21 +1,39 @@
+This profile sets minimum expectations for the [Specimen] resource to represent data requested as part of the U.S. Public Health Case Based Surveillance messaging guides. It identifies which core elements, extensions, vocabularies and value sets **SHALL** be present in the resource when using this profile.
+
+**Example Usage Scenarios:**
+
+The following are example usage scenarios for the US CBS Patient profile:
+
+-   Represent a specimen tested as part of a diagnostic lab report.
+
+### Relationship to ECR Public Health and US Core Conditions
```
Comment (astevensgtri) [inline input/pagecontent/StructureDefinition-cbs-specimen-intro.md:9]: Should this section be here

Comment (astevensgtri) [reply input/pagecontent/StructureDefinition-cbs-specimen-intro.md:9]: (its all about condition, not specimen)

**Review by astevensgtri — APPROVED** [review]
Only need to look at the Specimen intro since it mentions Condition

File: input/pagecontent/StructureDefinition-cbs-specimen-intro.md (hunk 1 above)
Comment (EllieShivers) [reply input/pagecontent/StructureDefinition-cbs-specimen-intro.md]: removed section, fixed other parts that say condition

**Pull request merged**